executor/aggfuncs/func_json_objectagg.go

```diff
@@ -0,0 +1,113 @@
+package aggfuncs
+
+import (
+	"fmt"
```

Comment:
please separate the standard libs and the third-part libs with an empty line, like this:
```go
"fmt"
"reflect"

"github.com/pingcap/errors"
...
```